bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -65,13 +65,17 @@
 
 		<channels>
 			<channel id="setpointTemperature" typeId="setpointTemperature"/>
-			<channel id="temperature" typeId="temperature"/>
+			<channel id="temperature" typeId="system.indoor-temperature"/>
```

Comment:
To preserve existing label and description:
```suggestion
			<channel id="temperature" typeId="system.indoor-temperature">
				<label>Zone Temperature</label>
				<description>Gets the temperature of this zone</description>
			</channel>
```

Otherwise label will be "Indoor Temperature" and description "Current indoor temperature". I don't know if this is intended.

Please check others as well.

Label and description must be included in update instructions as well when overriding channel type.

I18N properties must also be regenerated.

See:
https://github.com/openhab/openhab-core/blob/6bf59605bdc25f57ce3792467075a09c567d349c/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java#L332-L337

Reply:
Not sure why it didnt show up in my first tests. Re-tested and fixed.

Reply:
It might have been because you tested the update, which would set the overridden labels and description you already provided in the update instructions. When creating a new Thing, they wouldn't have been shown with these labels.